platformio.ini

```diff
@@ -16,15 +16,16 @@ default_envs =
   esp07
   huzzah
   d1_mini_pro
+  esp32doit-devkit-v1
 
 [base]
 framework = arduino
-platform = espressif8266@~4
-board_build.ldscript = eagle.flash.4m1m.ld
-lib_deps =
-  https://github.com/tzapu/WiFiManager.git#v2.0.16-rc.2
-  RF24@~1.3.2
-  ArduinoJson@~6.21
+board_f_cpu = 160000000L
+lib_deps_builtin =
+  SPI
+lib_deps_external =
+  nrf24/RF24 @ ^1.4.9
```

Comment:
I get compile errors for esp8266 using `nrf24/RF24` v1.4.9. If we need to be using 1.4.9 for esp32, let's split it into the platform-specific dependencies so we can keep the version at `~1.3.2` for ESP8266.

Reply:
For reference this is the compile error:

```
.pio\libdeps\d1_mini\RF24/RF24_config.h:189:26: error: 'class HardwareSerial' has no member named 'Serial'
```

Reply:
My bad, better to keep 1.3.2 for esp32 too :)